src/WattleScript.Interpreter/DataTypes/Range.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace WattleScript.Interpreter
+{
+    public class Range : RefIdObject
```

Comment:
This could use a ToString() implementation for debugger visualisation

Reply:
Added in [ea86dbd](https://github.com/WattleScript/wattlescript/pull/36/commits/ea86dbd442ce1abc6a2f39118fec572d5ccebfb3)